File: FPNg-UI/src/app/feature/display/shared/view-models/ledger-vm.ts
import { IItemVM } from './item-vm';

export interface ILedgerVM {
  rollupKey: number;
  year: number;
  wDate: Date;
  creditSummary: number;
  debitSummary: number;
  net: number;
  runningTotal: number;
  items: IItemVM[];
}

File: FPNg-API/FPNg.API.Infrastructure/Display/Models/LedgerVm.cs
using System;
using System.Collections.Generic;

namespace FPNg.API.Infrastructure.Display.Models
{
    public class LedgerVM
	{
        public LedgerVM()
        {
			Items = new List<ItemVM>();
		}
		public int RollupKey { get; set; }
		public int Year { get; set; }
		public DateTime WDate { get; set; }
		public double CreditSummary { get; set; }
		public double DebitSummary { get; set; }
		public double Net { get; set; }
		public double RunningTotal { get; set; }
		public List<ItemVM> Items { get; set; }
    }
}
